desktop/src/com/frostwire/gui/bittorrent/CreateTorrentDialog.java

```diff
@@ -213,6 +228,12 @@ private void initTorrentTracking() {
         ThemeMediator.fixKeyStrokes(_textWebSeeds);
         torrentTrackingPanel.add(new JScrollPane(_textWebSeeds), "gapright 5, gapleft 80, gapbottom 5, hmin 165px, growx 60, growy");
 
+        // Select piece size
+        _labelPieceSize = new JLabel(I18n.tr("Piece Size"));
+        torrentTrackingPanel.add(_labelPieceSize, "align left, pushy, gapleft 5");
```

Comment:
Elements aren't positioned yet, because i still don't discover how to use coordinates system in MigLayout.